cg/store/models.py

```diff
@@ -373,32 +373,32 @@ def __str__(self) -> str:
         return f"{self.internal_id} ({self.name})"
 
     @property
-    def samples(self) -> List[str]:
+    def samples(self) -> List["Sample"]:
```

Comment:
Why the `"Sample"`and not `Sample`?

Reply:
Since Sample has not yet been defined in this file (python reads top down), we cannot use Sample, however "Sample" gives the exact same functionality.